lib/config.js

```diff
@@ -102,7 +103,7 @@ function createConfigPage(pageId, currentConfig) {
             required: true,
             multiple: false,
             capabilities: ['colorControl', 'switch', 'switchLevel'],
-            permissions: ['r', 'x']
+            permissions: [ 'r:devices:*', 'x:devices:*' ]
```

Comment:
This part was correct before – I'm not sure if you've got appropriate access, but take a look at [the spec](https://codesamsung.com/migration/iot/librarian/blob/master/swagger/smartapps.st.internal/v1.yml#L746-754) for a `DeviceSetting`. 

If you can't log in to the repo, here's the YAML spec:

```yaml
permissions:
          description: The required permissions for the selected device(s).
          type: array
          items:
            type: string
            enum:
            - r
            - x
            - w
```